plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java

```diff
@@ -326,4 +333,30 @@ public ThriftMetastoreConfig setWriteStatisticsThreads(int writeStatisticsThread
         this.writeStatisticsThreads = writeStatisticsThreads;
         return this;
     }
+
+    @Config("hive.metastore.client.http.bearer-token")
```

Comment:
```suggestion
@Config("hive.metastore.http.client.bearer-token")
```

See above the way that `thrift` is a "prefix" in the property names corresponding for thrift:

https://github.com/trinodb/trino/blob/f5eddd80d34b2a90daf169273f083d081acbf603/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java#L113-L128